src/main/java/command/BanUsersCommand.java

```diff
@@ -0,0 +1,28 @@
+package command;
+
+import container.UserContainer;
+import file.UserToFileWriter;
+import model.User;
+import model.UserRole;
+import model.UserStatus;
+
+public class BanUsersCommand {
+    public void ban(String userName) {
+        try {
+            User user = UserContainer.findUserByUsername(userName);
+
+            if (user != null) {
```

Comment:
Do not return null from container. You can throw exception and do not make null check